AdobeUi/Ui/Component/Listing/Columns/ImagePreview.php

```diff
@@ -6,7 +6,7 @@
 
 declare(strict_types=1);
 
-namespace Magento\AdobeStockImageAdminUi\Ui\Component\Listing\Columns;
+namespace Magento\AdobeUi\Ui\Component\Listing\Columns;
```

Comment:
Only image preview with navigation (next/forward/close) should be moved to AdobeUi. Displaying image attributes and buttons should still be part of AdobeStockImageAdminUi functionality (extending AdobeUi)